core/src/com/fallenflame/game/LevelController.java

```diff
@@ -1,5 +1,498 @@
 package com.fallenflame.game;
 
+import com.badlogic.gdx.math.*;
+import com.badlogic.gdx.physics.box2d.*;
+import com.badlogic.gdx.utils.*;
+import com.fallenflame.game.physics.obstacle.Obstacle;
+import com.fallenflame.game.physics.obstacle.ObstacleCanvas;
```

Comment:
This is unnecessary, we do not have an ObstacleCanvas

Reply:
Nice catch, thanks!